src/app/profile/lot-manager/lot-manager.component.html

```diff
@@ -0,0 +1,17 @@
+<div class="lot-manager wrapper">
+  <form [formGroup]="lotForm" (ngSubmit)="onSubmit()" class="lot-manager-form">
+    <h2 class="lot-manager-title">Lot Manager</h2>
```

Comment:
`Lot Manager` is slightly confusing title for lot creation page, please change to `New Lot` title

Reply:
it will create and manage lot